workers/unity/Packages/io.improbable.gdk.tools/.CodeGenTemplate/CodeGenerationLib/CodeWriter/Scopes/LoopBlock.cs

```diff
@@ -0,0 +1,15 @@
+using System;
+
+namespace Improbable.Gdk.CodeGeneration.CodeWriter.Scopes
+{
+    /// <summary>
+    /// Scope with mandatory declaration for defining loops.
+    /// </summary>
+    public class LoopBlock : ScopeBody
```

Comment:
Loop = While?

Reply:
answered elsewhere but loop -> foreach/for/while